sdk/core/System.Net.ClientModel/tests/clients/OpenAIClient/OpenAIClient.cs

```diff
@@ -0,0 +1,93 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Net.ClientModel;
+using System.Net.ClientModel.Core;
+using System.Net.ClientModel.Core.Pipeline;
+using System.Net.ClientModel.Internal;
+using System.Text;
+using System.Threading;
+
+namespace OpenAI;
+
+public class OpenAIClient
```

Comment:
This illustrates what a functioning client might look like today using types from System.Net.ClientModel.